contracts/Peeranha.sol

```diff
@@ -635,6 +639,11 @@ contract Peeranha is IPeeranha, Initializable {
         return userContext.userRatingCollection.communityRatingForUser[user].userPeriodRewards[rewardPeriod].periodRating[communityId].ratingToReward;
     }
 
+    // only unitTest
+    function getWeekReward(uint16 rewardPeriod) external view override returns(int32) {
```

Comment:
Change all names where used word "Week". Replace with "Period". The period length may change in the future.